atlp-rwanda/octopus-bn-frontend
# #170947383 Users update profile

#### What does this PR do?
- This PR sets up user profile
#### Description of Task to be completed?
- Allow users to view/update their profile
- Allow user to upload image to the cloudinary
- Configure internationalization
#### How should this be manually tested?
- Clone the repo
- checkout this branch
- Run ```npm i```
- Run ``` npm run test```
- Run ``` npm run dev```
- And check out this URL: ```http://localhost:8080/profile```
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#170947383](https://www.pivotaltracker.com/story/show/170947383)
### Screenshots (if appropriate)

### Profile page with avatar 

<img width="1440" alt="Screen Shot 2020-05-07 at 19 19 15" src="https://user-images.githubusercontent.com/51243858/81325270-64469f00-9098-11ea-9259-d5d5c11d6d0c.png">

### Dashboard update profile

<img width="1440" alt="Screen Shot 2020-05-07 at 17 44 44" src="https://user-images.githubusercontent.com/51243858/81319622-36f5f300-9090-11ea-8456-57c689a2a93e.png">



#### Questions:
N/A
